Source/Core/DotSpatial.Projections/ProjectionInfo.cs

```diff
@@ -1096,6 +1115,18 @@ private void ParseProj4String(string proj4String)
 
                     case "proj":
 
+                        //check for coordinate system type. A data set's coordinate system is identified by the PROJCS keyword if the data
+                        //are in projected coordinates, by GEOGCS if in geographic coordinates, or by GEOCCS if in geocentric coordinates. 
+                        //https://www.earthdatascience.org/courses/earth-analytics/spatial-data-r/understand-epsg-wkt-and-other-crs-definition-file-types/
+                        if (value == "longlat")
+                        {
+                            CoordinateSystemType  = CoordinateSystemType.Geographic;
+                        }
+                        else
+                        {
+                            CoordinateSystemType  = CoordinateSystemType.Projected;
+                        }
+
                         if (value == "longlat")
```

Comment:
Please combine this  `if (value == "longlat")` with the `if (value == "longlat")` you added.

Reply:
done